src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/MercuryArticlePageObject.java

```diff
@@ -157,14 +186,193 @@ public void verifyCanonicalUrl() {
   public void verifyRepliesAreExpanded() {
     Assertion.assertTrue(checkIfElementOnPage(repliesContent.get(0)));
   }
-  
+
   public void verifySingleLinkedImageRedirect(int index) {
+    String methodName = "verifySingleLinkedImageRedirect";
     String currentUrl = driver.getCurrentUrl();
     singleImgLink.get(index).click();
     if (currentUrl.equals(driver.getCurrentUrl())) {
-      PageObjectLogging.log("verifySingleLinkedImageRedirect", "Redirection doesn't work", false);
+      PageObjectLogging.log(methodName, "Redirection doesn't work", false);
+    } else {
+      PageObjectLogging.log(methodName, "Redirection works", true);
+    }
+  }
+
+  public void verify25CommentsPerPage() {
+    String methodName = "verify25CommentsPerPage";
+    if (commentsList.size() == 25) {
```

Comment:
Could consider using Assertion.assertEquals(int, int, String) instead of if/else